Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY .mvn $APP_HOME/.mvn
 COPY mvnw $APP_HOME
 
 WORKDIR $APP_HOME
-RUN ./mvnw package -Dlicense.skip=true -DskipTests && rm -rf ~/.m2
+RUN --mount=type=cache,target=/root/.m2 ./mvnw package -Dlicense.skip=true -DskipTests
```

Comment:
how does Maven know to look at `root/.m2`?

Reply:
Check out https://maven.apache.org/settings.html

> `localRepository`: This value is the path of this build system's local repository. The default value is `${user.home}/.m2/repository`

The username is `root`, thus `/root/.m2`